.github/workflows/ci.yaml

```diff
@@ -0,0 +1,24 @@
+name: Run Python Tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Python 3
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
```

Comment:
I'm on python 3.8.6, can we use that?

Reply:
Good call, updated it here: https://github.com/thegridelectric/gw-scada-spaceheat-python/pull/4/commits/8c61cb27f5ee0010dda6f28f03a42f418c56d317